strings/xenotips.txt

```diff
@@ -37,3 +37,21 @@ You can filter out the Xenomorphs displayed in hive status by health, allowing y
 Each xeno has their own ‘tackle counter’ on a marine. The range to successfully tackle can be anywhere from two to six tackles based on caste. If a marine gets stunned or knocked over by other means it will reset everyone's tackle counters and they may get up!
 As a Xenomorph, the list of available tunnels is sorted by their distance to the player!
 As a Xenomorph, pay attention to what a marine is wearing. If they have no helmet, aiming for the head will be significantly more damaging, if they aren't wearing gloves, then think about going for their hands.
+Applying acid to flares makes them burn faster.
+Melting weapons, ammo or other valuables owned by marines is a good way to debilitate them.
+The secrete acid shortcut is (by default) Shift + C.
+Placing acid traps under map debris such as metal sheets, wooden planks or other semi-large to large items is good for catching marines by surprise.
+Burrower can place tunnels underneath tables or lockers.
+Remember that Lessers and Huggers cannot speak or use the Hivemind for three minutes after spawning.
+Alt-clicking on the Queen tracker allows you to set it to also track the Hive Core, any living leaders or any existing tunnels and not just the Queen. Tunnels are ordered by closest to furthest; top to bottom.
+Be mindful of not body-blocking the escape of any fleeing sisters; this is especially true if they are heavily injured.
```

Comment:
Don't particularly like how this is worded.

Reply:
all four lines or one specific one?